tests/tt_eager/python_api_testing/unit_testing/misc/test_unary_ops_ttnn.py

```diff
@@ -26,3 +27,66 @@ def test_unary_square_ttnn(input_shapes, device):
 
     comp_pass = compare_pcc([output_tensor], [golden_tensor])
     assert comp_pass
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
+@pytest.mark.parametrize("exponent", [0.5, 2])
+def test_unary_pow_ttnn(input_shapes, exponent, device):
```

Comment:
Why are tests being added here and not in `tests/ttnn/unit_tests/operations`?

Reply:
I agree. It is not perfect, but we have tons of tests in that folder already. 
We will have to move things after move all the ops. 
I recommend to move this later. 

But @KalaivaniMCW going forward, I definitely agree with @arakhmati and recommend to put things into ttnn in the first place.

@arakhmati would you agree to unblock merge?

Reply:
@arakhmati 
`tests/ttnn/unit_tests/operations/test_unary.py` already has tests for all these unary ops. 
The test file (.py file) changes are being done as part of another PR since we wanted a lighter PR for review. 
This file here`tests/tt_eager/python_api_testing/unit_testing/misc/test_unary_ops_ttnn.py` is for the purpose of this migration task to test any newly implemented op/structs in ttnn, and it is shared by many in our team for testing purpose. This file will probably be moved or modified at the final round of migration.